src/checkout/create-internal-checkout-selectors.ts

```diff
@@ -96,6 +99,7 @@ export function createInternalCheckoutSelectorsFactory(): InternalCheckoutSelect
             shippingAddress,
             shippingCountries,
             shippingStrategies,
+            subscriptions,
```

Comment:
Can you also export the new selectors you've created in `CheckoutStoreStatusSelector` and `CheckoutStoreErrorSelector` so that they can be called externally?